### DISCUSSION TITLE
Last 3 weeks I'm getting every day Error "Timed out 600 seconds" in Full Node Status.

### DISCUSSION THREAD

**Discussion by RichardRozehnal** [Chia-Network/chia-blockchain#17762] [Support]
Hello,
i had this problem in december 2023 but after then it was OK until now.
I'm getting error "Timed out 600 seconds" in Full Node Status every day. I updated to version 2.2.1 and the error still occurs. After this error i have to restart GUI and then synchronize but next day i get the error again.
I farm on RPi 4 4GB RAM.

How can i fix it?

thank you

**Comment by BrandtH22** [comment]
Hey @RichardRozehnal , as a heads up we generally can provide more timely and thorough support in our discord server https://discord.gg/chia . 

Since around that time you indicated there has been an influx of transactions which makes it a bit more difficult for the 4GB rpi to keep up. There are a few settings that can be updated in the config file to help with this and we are happy to work through those with you in our discord server :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
